llm/vllm/serve-openai-api.yaml

```diff
@@ -10,17 +10,13 @@ resources:
 setup: |
   conda activate vllm
   if [ $? -ne 0 ]; then
-    conda create -n vllm python=3.9 -y
+    conda create -n vllm python=3.10 -y
     conda activate vllm
   fi
 
-  git clone https://github.com/vllm-project/vllm.git || true
-  # Install fschat and accelerate for chat completion
-  pip install fschat
-  pip install accelerate
+  pip list | grep vllm || pip install vllm==0.3.0
+  pip install git+https://github.com/huggingface/transformers
```

Comment:
should we specify a version/commit hash? I'm not sure if it is robust enough since the master branch is accepting new commits everyday?

Reply:
Good point! Pinned the version for transformers.